debian/copyright

```diff
@@ -133,14 +133,51 @@ Files: test-run/lib/msgpack-python/*
 Copyright: 2008-2011, INADA Naoki <songofacandy@gmail.com>
 License: Apache-2.0
 
+Files: test-run/lib/checks/*
+Copyright: 2010-2018 Google, Inc
+License: MIT
```

Comment:
Wow. Someone blindly copies the license.

Reply:
Can you, please, fix it in checks and then there?

Reply:
Wow, it is the derived work (I didn't know). I filed https://github.com/fab13n/checks/issues/2 to ask the original authors, whether they're okay with the MIT license.

Reply:
Should be resolved by https://github.com/tarantool/checks/pull/28

Reply:
Fixed